src/ypd/form/project_form.py

```diff
@@ -1,18 +1,34 @@
 from flask_wtf import FlaskForm
-from wtforms import (BooleanField, IntegerField, RadioField, StringField, SubmitField, TextAreaField)
+from wtforms import (BooleanField, IntegerField, RadioField, StringField, SubmitField, TextAreaField, SelectMultipleField)
 from wtforms.validators import InputRequired, Length
-from wtforms.widgets import TextArea
+from wtforms.widgets import TextArea, CheckboxInput, ListWidget
+from ypd.model.project import GradeAttributes, DegreeAttributes
 
 class SubmissionForm(FlaskForm):
     """Submission Form"""
     PROVIDED = 0
     SOLICITED = 1
 
-    title = StringField('Title:', validators=[InputRequired()])
+    
+
+    title = StringField('Title:', validators=[InputRequired()], widget=TextArea(), render_kw={'cols': '150', 'rows': '2'})
     description = TextAreaField('Project Summary:', validators=[InputRequired()],
                                 widget=TextArea(), render_kw={'cols': '150', 'rows': '25'})
     projType = RadioField('Project Type:', validators=[InputRequired()], coerce=int,
                           choices=[(PROVIDED, 'Provided Project'), (SOLICITED,'Solicited Project')])
+    maxProjSize = IntegerField("Maximum Number of People Needed: ")
```

Comment:
If I leave the MaxProjSize box blank, It gets set to a null value in the database. We should flash an error to the user instead

Reply:
Set it so that if left blank, sets maxProjSize to 1

Reply:
Flashes error if no entry